data/tokens.json

```diff
@@ -3407,6 +3351,63 @@
         "textCase": "{gtmds_text-case-none}"
       },
       "type": "typography"
+    },
+    "gtmds_popover_margin": {
+      "value": "{gtmds_space-xs}",
+      "type": "spacing"
+    },
+    "gtmds_popover-header_margin-horizontal": {
+      "value": "{gtmds_space-xxs}",
```

Comment:
```suggestion
      "value": "{gtmds_size-md}",
```

Reply:
The space between the X and the Title becomes big, which limits a lot the space for the subtitle as well. Let's talk